TODO/using-a-function-in-setstate-instead-of-an-object.md

```diff
@@ -67,8 +101,14 @@ submit(){
 
 Passing in a function into `setState` instead of an object will give you a reliable value for your component’s `state` and `props`. One thing to note is that the React documentation makes use of arrow functions in their examples (which is also on my list of things to migrate to in my Shopsifter app!) so in my example above I’m using ES5 syntax for my function.
 
+通过使用函数替代对象传入 `setState` 的方式能够得到组件的 `state` 和 `props` 属性可靠的值。值得注意的一点是，在 React 文档的例子中使用了箭头函数（这也是我将要应用到我的 Shopsifter 应用中的一项内容），因此上文的例子中我的函数使用的仍然是 ES5 的语法。
+
 If you know you’re going to use `setState` to update your component and you know you’re going to need the current state or the current props of your component to calculate the next state, passing in a function as the first parameter of `this.setState` instead of an object is the recommended solution.
 
+如果你知道自己将要使用 `setState` 来更新组件，并且你知道自己将要使用当前组件的状态或者属性值来计算下一个状态，我推荐你传入一个函数作为 `this.setState` 的第一个参数而不用数组的解决方案。
```

Comment:
> passing in a function as the first parameter of `this.setState` instead of an object is the recommended solution.
> 我推荐你传入一个函数作为 `this.setState` 的第一个参数而不用数组的解决方案。

**笔误：** 数组 => 对象